pkg/utils/syscalls/syscalls_amd64.go

```diff
@@ -0,0 +1,738 @@
+	157: "prctl",
+	17:  "pread64",
+	295: "preadv",
+	327: "preadv2",
+	302: "prlimit64",
+	440: "process_madvise",
+	448: "process_mrelease",
+	310: "process_vm_readv",
+	311: "process_vm_writev",
+	270: "pselect6",
+	101: "ptrace",
+	18:  "pwrite64",
+	296: "pwritev",
+	328: "pwritev2",
+	178: "query_module",
+	179: "quotactl",
+	443: "quotactl_fd",
+	0:   "read",
+	187: "readahead",
+	89:  "readlink",
+	267: "readlinkat",
+	19:  "readv",
+	169: "reboot",
+	45:  "recvfrom",
+	299: "recvmmsg",
+	47:  "recvmsg",
+	216: "remap_file_pages",
+	197: "removexattr",
+	82:  "rename",
+	264: "renameat",
+	316: "renameat2",
+	249: "request_key",
+	219: "restart_syscall",
+	84:  "rmdir",
+	334: "rseq",
+	13:  "rt_sigaction",
+	127: "rt_sigpending",
+	14:  "rt_sigprocmask",
+	129: "rt_sigqueueinfo",
+	15:  "rt_sigreturn",
+	130: "rt_sigsuspend",
+	128: "rt_sigtimedwait",
+	297: "rt_tgsigqueueinfo",
+	146: "sched_get_priority_max",
+	147: "sched_get_priority_min",
+	204: "sched_getaffinity",
+	315: "sched_getattr",
+	143: "sched_getparam",
+	145: "sched_getscheduler",
+	148: "sched_rr_get_interval",
+	203: "sched_setaffinity",
+	314: "sched_setattr",
+	142: "sched_setparam",
+	144: "sched_setscheduler",
+	24:  "sched_yield",
+	317: "seccomp",
+	23:  "select",
+	66:  "semctl",
+	64:  "semget",
+	65:  "semop",
+	220: "semtimedop",
+	40:  "sendfile",
+	307: "sendmmsg",
+	46:  "sendmsg",
+	44:  "sendto",
+	238: "set_mempolicy",
+	450: "set_mempolicy_home_node",
+	273: "set_robust_list",
+	205: "set_thread_area",
+	218: "set_tid_address",
+	171: "setdomainname",
+	123: "setfsgid",
+	122: "setfsuid",
+	106: "setgid",
+	116: "setgroups",
+	170: "sethostname",
+	38:  "setitimer",
+	308: "setns",
+	109: "setpgid",
+	141: "setpriority",
+	114: "setregid",
+	119: "setresgid",
+	117: "setresuid",
+	113: "setreuid",
+	160: "setrlimit",
+	112: "setsid",
+	54:  "setsockopt",
+	164: "settimeofday",
+	105: "setuid",
+	188: "setxattr",
+	30:  "shmat",
+	31:  "shmctl",
+	67:  "shmdt",
+	29:  "shmget",
+	48:  "shutdown",
+	131: "sigaltstack",
+	282: "signalfd",
+	289: "signalfd4",
+	41:  "socket",
+	53:  "socketpair",
+	275: "splice",
+	4:   "stat",
+	137: "statfs",
+	332: "statx",
+	168: "swapoff",
+	167: "swapon",
+	88:  "symlink",
+	266: "symlinkat",
+	162: "sync",
+	277: "sync_file_range",
+	306: "syncfs",
+	139: "sysfs",
+	99:  "sysinfo",
+	103: "syslog",
+	276: "tee",
+	234: "tgkill",
+	201: "time",
+	222: "timer_create",
+	226: "timer_delete",
+	225: "timer_getoverrun",
+	224: "timer_gettime",
+	223: "timer_settime",
+	283: "timerfd_create",
+	287: "timerfd_gettime",
+	286: "timerfd_settime",
+	100: "times",
+	200: "tkill",
+	76:  "truncate",
+	95:  "umask",
+	166: "umount2",
+	63:  "uname",
+	87:  "unlink",
+	263: "unlinkat",
+	272: "unshare",
+	134: "uselib",
+	323: "userfaultfd",
+	136: "ustat",
+	132: "utime",
+	280: "utimensat",
+	235: "utimes",
+	58:  "vfork",
+	153: "vhangup",
+	278: "vmsplice",
+	61:  "wait4",
+	247: "waitid",
+	1:   "write",
+	20:  "writev"
```

Comment:
missing `,` at the end of the line: "pkg/utils/syscalls/syscalls_amd64.go:737:15: syntax error: unexpected newline in composite literal; possibly missing comma or }"

Reply:
I have the same error

Reply:
my linter missed it...